streams-runtimes/streams-runtime-local/src/main/java/org/apache/streams/local/tasks/StreamsPersistWriterTask.java

```diff
@@ -146,7 +146,7 @@ public void run() {
             try {
                 Thread.sleep(5000);
             } catch (InterruptedException e) {
-                LOGGER.error("Sleep interrupted: {}", e);
+                //This is the expected behavior when this thread is killed
```

Comment:
In lieu of refactoring, I'd still log this as info or debug so it's available if we need it.